### DISCUSSION TITLE
Missing Undock / Dock button when only one MindMap is open within the program.

### DISCUSSION THREAD

**Discussion by xhosch** [freeplane/freeplane#2073] [Help]
With the freeplane program open and only one MindMap open, I cannot open the MindMap as a “second (external) window”. I therefore always open a second MindMap within the program so that I can then undock the first MindMap. Have I overlooked another option?
Many thanks and best regards
Holger

**Comment by euu2021** [comment]
I don't see any native way of doing it. It's only possible using scripts.

What do you want to achieve? A windowed fullscreen view? Would it be better to have a script that toggles between two states in the main window: with bars (menubar, toolbar etc); without bars?

**Comment by dpolivaev** [comment]
You might have some very special use case, please explain why you want to undock a single mind map.
If only one mind map is shown, the tab is intentionally hidden to better use limited screen space and to reduce distractions.

**Reply by xhosch** [reply]
Hello and thank you very much for your quick replies.
My use case is as follows:
I use freeplane as a tool for teaching purposes. For me, it is a real alternative to conventional PDF/PPT presentations. 
I have an external screen connected to my notebook which shows me the detached mind map. This external screen is mirrored on the projector or large monitor for the training participants.
Additional information / keywords are displayed on the screen of my notebook, which is not visible to the training participants and which I only communicate verbally to the training participants.
Separating the MindMap and the window with the notes is only possible if I have previously opened a second new or different MindMap.
All in all, not a big problem. I'm just surprised that I have to separate the two windows in a somewhat cumbersome way, even though there would surely be a simpler solution.
Many thanks and best regards
Holger

**Reply by euu2021** [reply]
I don't know if it's helpful, but you can open a new map view (that is like a new tab of the same map), and then undock it. View->New map view (Ctrl+T). This way, you don't need to create a new map.

**Reply by xhosch** [reply]
Hello and yes, this is exactly what I am currently doing. That is of course no problem at all. Nevertheless, it is a bit strange to have to proceed in this way. :-)
Many thanks and best regards
Holger

**Comment by Illegit** [comment]
May I suggest utilizing a second view of the same map?

**Reply by xhosch** [reply]
Of course, this is also possible. :-)
